pkg/scheduler/framework/plugins/podtopologyspread/pod_topology_spread.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package podtopologyspread
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
+)
+
+type PodTopologySpread struct{}
+
+var _ = framework.FilterPlugin(&PodTopologySpread{})
```

Comment:
nit: This a more common for doing this, I updated the rest in https://github.com/kubernetes/kubernetes/pull/84073

```suggestion
var _ framework.FilterPlugin = &PodTopologySpread{}
```

```go
$ ag 'var _' pkg
pkg/volume/configmap/configmap.go
48:var _ volume.VolumePlugin = &configMapPlugin{}
142:var _ volume.Volume = &configMapVolume{}
159:var _ volume.Mounter = &configMapVolumeMounter{}
333:var _ volume.Unmounter = &configMapVolumeUnmounter{}

pkg/volume/rbd/attacher.go
89:var _ volume.Attacher = &rbdAttacher{}
91:var _ volume.DeviceMounter = &rbdAttacher{}
193:var _ volume.Detacher = &rbdDetacher{}
195:var _ volume.DeviceUnmounter = &rbdDetacher{}

pkg/volume/git_repo/git_repo.go
42:var _ volume.VolumePlugin = &gitRepoPlugin{}
145:var _ volume.Volume = &gitRepoVolume{}
164:var _ volume.Mounter = &gitRepoVolumeMounter{}
277:var _ volume.Unmounter = &gitRepoVolumeUnmounter{}

pkg/volume/rbd/rbd_util.go
275:var _ diskManager = &RBDUtil{}

pkg/volume/rbd/rbd.go
58:var _ volume.VolumePlugin = &rbdPlugin{}
59:var _ volume.PersistentVolumePlugin = &rbdPlugin{}
60:var _ volume.DeletableVolumePlugin = &rbdPlugin{}
61:var _ volume.ProvisionableVolumePlugin = &rbdPlugin{}
62:var _ volume.AttachableVolumePlugin = &rbdPlugin{}
63:var _ volume.ExpandableVolumePlugin = &rbdPlugin{}
64:var _ volume.BlockVolumePlugin = &rbdPlugin{}
65:var _ volume.DeviceMountableVolumePlugin = &rbdPlugin{}
220:var _ volume.NodeExpandableVolumePlugin = &rbdPlugin{}
611:var _ volume.Provisioner = &rbdVolumeProvisioner{}
753:var _ volume.Deleter = &rbdVolumeDeleter{}
779:var _ volume.Volume = &rbd{}
824:var _ volume.Mounter = &rbdMounter{}
861:var _ volume.Unmounter = &rbdUnmounter{}
885:var _ volume.BlockVolumeMapper = &rbdDiskMapper{}
899:var _ volume.BlockVolumeUnmapper = &rbdDiskUnmapper{}
...
```

Reply:
SG.